EMaligner/EMaligner.py

```diff
@@ -442,7 +428,7 @@ def create_CSR_A(self, resolved):
         tile_ids = np.array([t.tileId for t in resolved.tilespecs])
         fargs = [[pairs[i], i, self.args, tile_ids] for i in range(npairs)]
 
-        with renderapi.client.WithPool(self.args['n_parallel_jobs']) as pool:
+        with stdlib_pool.WithThreadPool(self.args['n_parallel_jobs']) as pool:
```

Comment:
if you were to (earlier on) do `renderapi.client.WithPool = stdlib_pool.WithThreadPool`, it should also switch to using threadpools for everything in the renderapi client (e.g. `import_tilespecs_parallel`).  I don't think that has a performance hit and might make a difference on Windows clients.

Reply:
did this. `import_tilespecs_parallel` still switches into pathos if in python 2 and trying to suppress render output.